db/scripts/01_cassandra_queue.cql

```diff
@@ -13,7 +14,7 @@ CREATE TABLE queue (
 );
 
 CREATE TABLE queue_size (
```

Comment:
i thought you could have multiple columns, as long as the column is a counter column.